docs/pages/versions/unversioned/guides/assets.md

```diff
@@ -20,4 +20,10 @@ Each time you publish your app, Expo will upload your assets to Amazon CloudFron
 
 ## Performance
 
+### Fonts
+
 Some assets are too important to start your app without. Fonts often fall into this category. On the web the font loading problem is known by several acronyms: FOUT, FOIT, and FOFT, which stand for Flash of Unstyled Text, Flash of Invisible Text, and Flash of Faux Text ([read more here](https://css-tricks.com/fout-foit-foft/)). The default behaviour with the icon-font-powered [@expo/vector-icons](../icons/#icons) icons is a FOIT on first load, and on subsequent loads the font will be automatically cached. Users have higher standards for mobile than web, so you might want to take it a step further by preloading and caching the font and important images during the initial loading screen.
+
+### Images
+
+Images often take up the most space out of the assets in an Expo project. Optimizing your images will make them take up less space on end users' devices and reduce the time and bandwidth needed to download before they are ready to display. To compress the images (PNGs and JPGs) in your project, you can run `expo optimize`. Alternatively, when you publish using `expo publish`, you will be prompted and asked if you'd like to do so if you haven't already. You can pass in a `--save` flag to backup a copy of each file with a `.orig` extension so that you can easily restore them if you find you don't like the compressed version. To compress the images to a certain quality you can pass in a `--quality=N` flag where `N` is an integer between 1 and 100 inclusive. Otherwise, running the command will compress the image to 60% by default. You can compress specific assets using `--include="[pattern]"`. This will only optimize assets that match this glob pattern. Otherwise, it defaults to using the `assetBundlePatterns` field in your `app.json`. If you want to exclude certain assets you can pass in `--exclude="[pattern]"`. This will avoid assets that match this glob pattern. Glob patterns are always relative to project root regardless of where you call the command from.
```

Comment:
Sorry to keep nitpicking here, but I still think this could be improved. It's a pretty big block of text right now and people generally have an easier time parsing text in smaller chunks (and are more likely to actually read it!). I'd suggest breaking this into 2 or 3 paragraphs, and/or putting all of the flags in a bulleted list rather than in prose.

Reply:
No worries, I appreciate the scrutiny. I want to make sure this looks good too so feel free to keep doing so. I addressed these concerns in 739470b and it's looking much better. Let me know if I need to change anything else.